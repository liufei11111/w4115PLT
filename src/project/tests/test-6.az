Float f1;
Float f2;
Float f3;
Int main(Int argc, String argv)｛
f1 = 1;
f1 = 0.2;
f1 = 1e10;
f1 = 2E15;
f1 = 2e-9;
f1 = 2e+12;
f1 = 2E-0;
f1 = 2E+12;

f2 = f1 + 1;
f2 = f1 + 1.5;
f2 = 1.5 + f1;
f2 = f1 + 19e-10;
f2 = 1.5 + 19e10;

f2 = 9 - f1;
f2 = 10 - 9.3;
f2 = 9e10 - 8E+2;

f3 = f1 * f2;
f3 = f1 * 2;
f3 = f1 * 10e12;
f3 = 10e12 * f1;
f1 = 3e-19 * 1.5;

f1 = f2 / 2;
f1 = f2 /1.5;
f1 = 1.5 / f2;
f1 = f2 / 9e10;
f1 = 9e10 / 5E2;
f1 = 9e10 / 1.6;
｝

