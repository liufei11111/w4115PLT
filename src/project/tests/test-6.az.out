stmts:

float f1;
float f2;
float f3;
f1 = 1;
f1 = 0.2;
f1 = 10000000000.;
f1 = 2e+15;
f1 = 2e-09;
f1 = 2e+12;
f1 = 2.;
f1 = 2e+12;
f2 = f1 + 1;
f2 = f1 + 1.5;
f2 = 1.5 + f1;
f2 = f1 + 1.9e-09;
f2 = 1.5 + 190000000000.;
f2 = 9 - f1;
f2 = 10 - 9.3;
f2 = 90000000000. - 800.;
f3 = f1 * f2;
f3 = f1 * 2;
f3 = f1 * 1e+13;
f3 = 1e+13 * f1;
f1 = 3e-19 * 1.5;
f1 = f2 / 2;
f1 = f2 / 1.5;
f1 = 1.5 / f2;
f1 = f2 / 90000000000.;
f1 = 90000000000. / 500.;
f1 = 90000000000. / 1.6;

fdecls:



